libbeat/keystore/file_keystore.go

```diff
@@ -245,11 +244,11 @@ func (k *FileKeystore) doSave(override bool) error {
 		return fmt.Errorf("cannot open file to save the keystore to '%s', error: %s", k.Path, err)
 	}
 
-	f.Write(version)
+	_, _ = f.Write(version)
 	base64Encoder := base64.NewEncoder(base64.StdEncoding, f)
-	io.Copy(base64Encoder, encrypted)
+	_, _ = io.Copy(base64Encoder, encrypted)
 	base64Encoder.Close()
-	f.Sync()
+	_ = f.Sync()
```

Comment:
Can we at least return these errors, I know it was not there before but should we suppress them in this function? This does not feel write, I think it was a bug.

Reply:
`f.Sync` errors are not returned on purpose. Those might fail sometimes, but it is not an error from Beats point of view. Also,  if the write calls return an error it is not an issue, because Beat saves the files constantly, so we are not losing data/information.

Unfortunately, when we touch the disk things can go wrong more times than we think. So we have resilience at strategically important file writes (see. `SafeFileRotate`).  But not all `Write` call has to be checked.

Reply:
@kvch could you please turn your reply into a comment before those suppressed errors? otherwise it looks strange that everyone is fine with it.